main.py

```diff
@@ -16,6 +17,20 @@
 
 PackageManifest.update_forward_refs()
 app = FastAPI()
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=[
+        # NOTE: When running `npm run dev`, this is the URL
+        "http://localhost:5173",
+        # NOTE: When running `npm run build && npm run preview`, this is the URL
+        "http://localhost:4173",
+        # NOTE: This is where the UI gets hosted
+        "https://remix.ethereum.org",
```

Comment:
could we also add
https://remix-alpha.ethereum.org
https://remix-beta.ethereum.org
package://6fd22d6fe5549ad4c4d8fd3ca0b7816b.mod (this is for the desktop version)
thanks!

Reply:
```suggestion
        "https://remix.ethereum.org",
        "https://remix-alpha.ethereum.org",
        "https://remix-beta.ethereum.org",
```